app/views/layouts/_navbar.html.erb

```diff
@@ -12,7 +12,7 @@
         </li>
         <% if request.path == building_map_path %>
           <li class="nav-item">
-            <a class="nav-link" data-bs-toggle="offcanvas" data-bs-target="#offcanvas"> <i class="fas fa-directions" data-bs-toggle="offcanvas" data-bs-target="#offcanvas"></i> Navigation </a>
+            <a class="nav-link" data-bs-toggle="offcanvas" data-bs-target="#offcanvas" id="nav-link-navigation"> <i class="fas fa-directions" data-bs-toggle="offcanvas" data-bs-target="#offcanvas"></i> Navigation </a>
```

Comment:
Warning: Everything regarding the nav bar will be removed since we won't have one once switching to a one page setup. But I guess it's our problem of adjusting those tests then 😅

Reply:
This one is just needed for our test to click this "not Button".